### PR TITLE
Master scrolling many2xxx dropdown jum

### PR DESCRIPTION
**Pull request by Arcasias**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Arcasias** [comment]
@VincentSchippefilt review pls

File: addons/web/static/tests/fields/basic_fields_tests.js [hunk 1]
```diff
@@ -2658,6 +2658,33 @@ QUnit.module('basic_fields', {
...
+                        '<field name="date"/>' +
+                    '</div>' +
+                '</form>',
+            res_id: 1,
+        });
+
+        await testUtils.form.clickEdit(form);
+        await testUtils.dom.openDatepicker(form.$('.o_datepicker'));
+        
+        assert.ok($('.bootstrap-datetimepicker-widget').length, "datepicker should be opened");
```
Comment (VincentSchippefilt) [inline addons/web/static/tests/fields/basic_fields_tests.js]: use containsOnce

File: addons/web/static/tests/fields/basic_fields_tests.js [hunk 2]
```diff
@@ -2658,6 +2658,33 @@ QUnit.module('basic_fields', {
...
+
+        form.el.dispatchEvent(new Event('scroll'));
+        assert.ok(! $('.bootstrap-datetimepicker-widget').length, "datepicker should be closed");
```
Comment (VincentSchippefilt) [inline addons/web/static/tests/fields/basic_fields_tests.js]: use containsNone

**Comment by VincentSchippefilt** [comment]
robodoo r+

**Pull request closed without merging**